common/src/main/scala/org/apache/comet/CometConf.scala

```diff
@@ -374,20 +374,22 @@ object CometConf extends ShimCometConf {
     .booleanConf
     .createWithDefault(COMET_SCHEMA_EVOLUTION_ENABLED_DEFAULT)
 
-  val COMET_ROW_TO_COLUMNAR_ENABLED: ConfigEntry[Boolean] =
-    conf("spark.comet.rowToColumnar.enabled")
+  val COMET_SPARK_TO_COLUMNAR_ENABLED: ConfigEntry[Boolean] =
+    conf("spark.comet.sparkToColumnar.enabled")
       .internal()
-      .doc("Whether to enable row to columnar conversion in Comet. When this is turned on, " +
-        "Comet will convert row-based operators in " +
-        "`spark.comet.rowToColumnar.supportedOperatorList` into columnar based before processing.")
+      .doc(
+        "Whether to enable row to columnar conversion in Comet. When this is turned on, " +
+          "Comet will convert row-based operators in " +
+          "`spark.comet.sparkToColumnar.supportedOperatorList` into columnar based before " +
+          "processing.")
```

Comment:
The documentation should also be updated (it still says this is for converting row to columnar only)

Reply:
Good catch, only did the find replaces